packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+const headers = ['Date and Time', 'Action', 'Browser', 'IP Address', 'Status'] as const;
+const columns = headers.map(header => ({
+    header,
+}));
+
+export const LoginHistory = () => {
+    return (
+        <div className='flex flex-col '>
+            <Table
+                columns={columns}
+                data={[
+                    {
+                        action: 'Login',
+                        browser: 'Chrome v122.0.0.0',
+                        datetime: '2024-03-07 12:04:42 GMT',
+                        ipAddress: '942',
+                        status: 'Successful',
+                    },
+                ]}
+                isFetching={true}
+                renderHeader={header => <span>{header}</span>}
+                rowRender={(data: {
+                    action: string;
+                    browser: string;
+                    datetime: string;
+                    ipAddress: string;
+                    status: string;
+                }) => (
```

Comment:
```suggestion
  const TRowData = {
                    action: string;
                    browser: string;
                    datetime: string;
                    ipAddress: string;
                    status: string;
                };
                ...
                
                rowRender={(data: TRowData) => (
```

Reply:
let's write the types ourside the component to be consistent and for maintainability

Reply:
![Screenshot 2024-03-11 at 11 59 12 AM](https://github.com/binary-com/deriv-app/assets/158162395/87e765b3-e065-4bbc-a38c-7d87d28b30a9)

the mentioned method gives and error